artio-core/src/main/java/uk/co/real_logic/artio/library/OnMessageInfo.java

```diff
@@ -30,7 +31,13 @@ public OnMessageInfo status(final MessageStatus status)
 
     public OnMessageInfo isValid(final boolean valid)
     {
-        isValid = valid;
```

Comment:
Please don't rename variables unrelated to the change in question, especially when it reduces the consistency of variable naming.